packages/dom/src/clone-dom.js

```diff
@@ -13,11 +14,12 @@ import applyElementTransformations from './transform-dom';
 
 const ignoreTags = ['NOSCRIPT'];
 
-export function cloneNodeAndShadow({ dom, disableShadowDOM }) {
+export function cloneNodeAndShadow({ dom, disableShadowDOM, resources }) {
   // clones shadow DOM and light DOM for a given node
   let cloneNode = (node, parent) => {
     let walkTree = (nextn, nextp) => {
       while (nextn) {
+        serializeBase64(nextn, resources);
```

Comment:
This is changing the node in original test browser. We want to update cloned node so that client test dom is not affected. Please move this under like 36 as previously discussed on call.

Reply:
updated